src/exercises/subobject-bounds/README.md

```diff
@@ -0,0 +1,92 @@
+# Explore Subobject Bounds
+
+In the CheriABI run-time environment, bounds are typically associated with
```

Comment:
I wonder if we mean "In CHERI C/C++" rather than CheriABI per se? This also applies to bare-metal code, the pure-capability kernel, etc...